src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachCaseCallbackService.java

```diff
@@ -416,6 +420,16 @@ private void updateSupplementaryEvidence(
         CaseDetails exceptionRecordDetails,
         boolean ignoreWarnings
     ) {
+        Optional<String> attachedToCase = getCaseExceptionRecordIsAttachedTo(
+            callBackEvent.exceptionRecordJurisdiction,
+            callBackEvent.exceptionRecordId
+        );
+
+        if (attachedToCase.isPresent()) {
+            log.warn("Exception record is already attached to case {}", attachedToCase.get());
```

Comment:
Suggestion: this warning is for us, so it would be ideal to make it something like:
```
There's been an attempt to attach an already attached exception record to another case. Exception record ID: {}, attempt to attach to case: {}, already attached to case: {} 
```
or something like this, so that we can see what's going on.

Reply:
Have done